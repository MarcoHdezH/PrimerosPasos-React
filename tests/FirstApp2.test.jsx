
const { render, getByText, getByTestId, fireEvent, screen} = require("@testing-library/react");
const { FirstApp } = require("../src/FirstApp");

describe('Pruebas en FirstApp', () => {

    const title='Hola Soy Goku';

    test('Match con SnapShot',()=>{
        const { container } = render( <FirstApp title={title}/>);
        expect(container).toMatchSnapshot();
    });

    test('debe de mostrar el mensaje "Hola Soy Goku" ',()=>{

        render(<FirstApp title={title}/>);
    });
});
