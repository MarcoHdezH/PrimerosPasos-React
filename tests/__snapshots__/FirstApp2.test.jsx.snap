// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en FirstApp Match con SnapShot 1`] = `
<div>
  <div
    data-testid="test-title"
  >
    Hola Soy Goku
  </div>
  <p>
    Recuerda poner el titulo amiguito(a)
  </p>
  <p>
    Falto tu nombre wey
  </p>
</div>
`;
